datajunction-server/datajunction_server/api/helpers.py

```diff
@@ -381,9 +364,12 @@ def validate_node_data(  # pylint: disable=too-many-locals
     Dict[NodeRevision, List[ast.Table]],
     Dict[str, List[ast.Table]],
     List[str],
+    List[DJError],
 ]:
     """
-    Validate a node.
+    Validate a node. This function should never raise any errors.
```

Comment:
🎉 yay, glad this change came in

Reply:
Thanks, I think I missed one little Exception in there but we can address it later.